wrangler-storage/src/main/java/io/cdap/wrangler/store/recipe/RecipeStore.java

```diff
@@ -89,37 +89,61 @@ public RecipeStore(TransactionRunner transactionRunner) {
    * @param recipeRow recipe to create/update
    */
   public void saveRecipe(RecipeId recipeId, RecipeRow recipeRow) {
```

Comment:
CreateRecipe would be a better name since save can point to both create and update.